.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Report a bug for the Python/testing components of detection-rules
+title: "[Bug] "
+labels: ["bug", "Team: TRADE"]
+assignees: []
+projects: ["elastic/1268"]
+
+body:
+  - type: textarea
+    id: bug_description
+    attributes:
+      label: Describe the Bug
+      description: "A clear and concise description of what the bug is."
+      placeholder: "Describe the bug..."
+
+  - type: textarea
+    id: reproduce_steps
+    attributes:
+      label: To Reproduce
+      description: "Steps to reproduce the behavior:"
+      placeholder: "1. Go to '...'\n2. Click on '....'\n3. Scroll down to '....'\n4. See error"
+
+  - type: textarea
+    id: expected_behavior
+    attributes:
+      label: Expected Behavior
+      description: "A clear and concise description of what you expected to happen."
+      placeholder: "Expected behavior..."
+
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: "If applicable, add screenshots to help explain your problem."
+      placeholder: "Upload screenshots..."
+
+  - type: dropdown
+    id: os
+    attributes:
+      label: Desktop - OS
+      options:
+      - Windows
+      - macOS
+      - Linux
```

Comment:
Makes sense to add the `other` option just in case. Im not sure which use cases would apply for mobile given our python codebase and usage. LMK if you have ideas here.

Reply:
Android would be applicable, there is a community Android agent that folks may want to use. https://github.com/swiftbird07/elastic-agent-android

Reply:
Gotcha. This is a python bug in the repo. Does this still make sense?

Reply:
TLDR: Probably does not make sense as an option. I think it is more effective to us the OS field for where the bug is occurring rather than what OS is effected. 

I may have been thinking of the role for this template incorrectly, but I was initially thinking that there could be a python bug with respect to code that causes a package/rule issue for Android rules. But in that case, while the bug could not be specific to that OS the actual logical issue would not be occurring on Android.

Reply:
I was a little confused too. I'll add `other -explain` and in the `Additional Context` input, people can explain there.